social/tests/backends/open_id.py

```diff
@@ -127,6 +131,29 @@ class OpenIdConnectTestMixin(object):
     client_key = 'a-key'
     client_secret = 'a-secret-key'
     issuer = None  # id_token issuer
+    openid_config_body = None
+    key = None
+
+    def setUp(self):
+        super(OpenIdConnectTestMixin, self).setUp()
+        here = os.path.dirname(__file__)
+        self.key = RSAKey(kid='testkey').load(os.path.join(here, '../testkey.pem'))
+        HTTPretty.register_uri(HTTPretty.GET,
+                               self.backend.OIDC_ENDPOINT + '/.well-known/openid-configuration',
```

Comment:
These long lines should likely be at or below 80 characters to match the rest of the file/app